docs/src/modules/components/HomeSteps.js

```diff
@@ -199,6 +199,7 @@ function HomeSteps(props) {
             data-ga-event-action="click"
             data-ga-event-label="home-image"
             className={classes.link}
+            aria-hidden="true"
```

Comment:
Why would we hide this link from assistive technology?

Reply:
I have seen the Wave Chrome extension report a similar adjacent issue on the ad template:

![Capture d’écran 2019-12-09 à 16 20 25](https://user-images.githubusercontent.com/3165635/70447954-e443eb00-1a9f-11ea-8507-a36b583cc0c3.png)

## Alerts - Redundant link

### What It Means

Adjacent links go to the same URL.

### Why It Matters

When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users.

### How to Fix It

If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").

### The Algorithm... in English

Two adjacent links go to the same URL.

### Standards and Guidelines

[2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)